src/components/roadmap-filter.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useRef, useMemo } from 'react'
+import Dropdown from './Dropdown'
+import Lists from '../utils/Lists'
+import Button from './Button'
+
+const RoadmapFilter = (props) => {
+  const { selectState, setPage, currentStateSelected } = props
+  const [status, setStatus] = useState('All')
+  const statusDropdownContainer = useRef()
+
+  const statusOptions = useMemo(() => {
+    const optionsArray = []
+    for (let i = 0; i < Lists.stateList.length; i++) {
+      optionsArray.push(Lists.stateList[i].status)
+    }
+    return optionsArray
+  }, [])
```

Comment:
Lists is constant right? if yes, why do we need useMemo then?

Reply:
No idea why this was introduced, to begin with. It was added in a UseEffect so I moved it to the constant as this is the correct usage.
But I agree with you, this is a static list for what I can see, so there is no need to useMemo(). I'll delete it